src/files.ts

```diff
@@ -266,21 +283,23 @@ export async function* collectBundleFiles(
 
 export async function prepareExtendingBundle(
   baseDir: string,
-  files: string[],
-  supportedFiles: ISupportedFiles,
+  supportedFiles: SupportedFiles,
   fileIgnores: string[] = IGNORES_DEFAULT,
+  files: string[],
   maxFileSize = MAX_PAYLOAD,
   symlinksEnabled = false,
-): Promise<{ files: IFileInfo[]; removedFiles: string[] }> {
+): Promise<{ files: FileInfo[]; removedFiles: string[] }> {
   let removedFiles: string[] = [];
-  let bundleFiles: IFileInfo[] = [];
+  let bundleFiles: FileInfo[] = [];
   const cache = new Cache(CACHE_KEY, baseDir);
 
   // Filter for supported extensions/files only
   let processingFiles: string[] = filterSupportedFiles(files, supportedFiles);
 
   // Exclude files to be ignored based on ignore rules. We assume here, that ignore rules have not been changed.
-  processingFiles = processingFiles.filter(f => !isMatch(f, fileIgnores));
+  processingFiles = processingFiles
+    .map(f => resolveBundleFilePath(baseDir, f))
```

Comment:
question: why is this extra step needed? Last time a change in the way we ignore things was made, we introduced  performance issues on the client.

Reply:
This should not bring performance degradation. However, the reason for the change is that in the previous api we were sending absolute paths in the bundles, while now we use relative paths instead (the consistency of the ignores should be guaranteed by the passing tests).